packages/hyperdrive-appconfig/src/appconfig/getAppConfig.ts

```diff
@@ -309,7 +310,7 @@ export async function getAppConfig({
   const configs: HyperdriveConfig[] = await Promise.all(
     hyperdrives.map(async (hyperdrive) => {
       const kind = await hyperdrive.getKind();
-      console.log("Hyperdrive:", kind, hyperdrive.address);
+      console.log(chalk.blue(kind), chalk.yellow(chainId), hyperdrive.address);
```

Comment:
Node 21 has this built-in now: https://x.com/mattpocockuk/status/1816427834606272965

```suggestion
      console.log(styleText("blue", kind), styleText("yellow", chainId), hyperdrive.address);
```

Reply:
Nice! It looks like we're on Node 20, but this is cool